gatsby/src/components/Prep/Prep.tsx

```diff
@@ -2,17 +2,33 @@ import { type FC } from "react";
 
 import { type FeedPrep } from "@nice-digital/gatsby-source-bnf";
 
+import { Accordion } from "@/components/Accordion/Accordion";
 import { type QueryResult } from "@/utils";
 
+import styles from "./Prep.module.scss";
+
 export interface PrepProps {
 	prep: QueryResult<FeedPrep>;
 }
 
-export const Prep: FC<PrepProps> = ({ prep }) => (
-	<>
-		<h3>
-			{prep.name} {prep.manufacturer} {prep.blackTriangle ? "\u25BC" : null}
-		</h3>
+export const Prep: FC<PrepProps> = ({ prep, children }) => (
+	<Accordion
+		title={
+			<h3 className={styles.prepHeading}>
+				<span className={styles.headingIcons}>
+					{prep.blackTriangle ? "\u25BC" : null}
+					{prep.sugarFree ? (
+						<span className={styles.sugarFree}>SUGAR FREE </span>
+					) : null}
+				</span>
+				<span className={styles.headingText}>
+					{prep.name}{" "}
+					<span className={styles.manufacturer}>{prep.manufacturer}</span>{" "}
```

Comment:
Doesn't look like we'd need this space on the end, now that the black triangle is first?